tests/test_registration_and_login.py

```diff
@@ -33,13 +33,17 @@ def test_user_registration_with_invite_code_enabled(client: FlaskClient) -> None
     os.environ["REGISTRATION_CODES_REQUIRED"] = "True"
 
     # Generate a valid invite code using the script
-    invite_code_str = create_invite_code()
+    code = secrets.token_urlsafe(16)
```

Comment:
I think we could refactor create_invite_code to be reusable here instead of re-implementing the functionality.

Reply:
I've moved the logic into the InviteCode constructor